src/service.ts

```diff
@@ -888,13 +901,8 @@ export const snowballRService: ISnowballR = {
             PROJECT_PAPERS.set(id, project_paper);
             PAPER_REVIEWS.set(id, []);
             PROJECT_PROJECT_PAPERS.get(projectId)!.push(id);
-            PROGRESS.set(projectId, Math.min(PROGRESS.get(projectId) || 0 + 0.05, 1.0));
+            PROGRESS.set(projectId, Math.min((PROGRESS.get(projectId) || 0) + 0.05, 1.0));
```

Comment:
Prefer using nullish coalescing operator (`??`) instead of a logical or (`||`), as it is a safer operator. ([`typescript:S6606`](https://rules.sonarsource.com/typescript/impact/maintainability/RSPEC-3512))
```suggestion
            PROGRESS.set(projectId, Math.min((PROGRESS.get(projectId) ?? 0) + 0.05, 1.0));
```